### DISCUSSION TITLE
Where is old discourse forum?

### DISCUSSION THREAD

**Discussion by dkebler** [quasarframework/quasar#9739] [Ideas / Proposals] [answered]
Looks like forum.quasar.dev now points here.   I guess this is an improvement????  Was the old discourse forum taken down?  If not what if we want to go back and look up some old threads?  How do we access it.

BTW how about a "general discussion" category for well, questions like this.

**Comment by rstoenescu** [comment, marked as answer]
https://forum.quasar-framework.org/

**Comment by hawkeye64** [comment]
@dkebler 
> BTW how about a "general discussion" category for well, questions like this.

GitHub only allows for 10 discussions and we are at the limit
